inc/class-storefront.php

```diff
@@ -125,42 +125,57 @@ public function setup() {
 		 *
 		 * @link http://codex.wordpress.org/Function_Reference/register_sidebar
 		 */
-		public function widgets_init() {
-			register_sidebar( array(
-				'name'          => __( 'Sidebar', 'storefront' ),
-				'id'            => 'sidebar-1',
-				'description'   => '',
-				'before_widget' => '<aside id="%1$s" class="widget %2$s">',
-				'after_widget'  => '</aside>',
-				'before_title'  => '<h3 class="widget-title">',
-				'after_title'   => '</h3>',
-			) );
-
-			register_sidebar( array(
-				'name'          => __( 'Below Header', 'storefront' ),
-				'id'            => 'header-1',
-				'description'   => 'Widgets added to this region will appear beneath the header and above the main content.',
-				'before_widget' => '<aside id="%1$s" class="widget %2$s">',
-				'after_widget'  => '</aside>',
-				'before_title'  => '<h3 class="widget-title">',
-				'after_title'   => '</h3>',
-			) );
-
-			$footer_widget_regions = apply_filters( 'storefront_footer_widget_regions', 4 );
-
-			for ( $i = 1; $i <= intval( $footer_widget_regions ); $i++ ) {
-				register_sidebar( array(
-					'name' 				=> sprintf( __( 'Footer %d', 'storefront' ), $i ),
-					'id' 				=> sprintf( 'footer-%d', $i ),
-					'description' 		=> sprintf( __( 'Widgetized Footer Region %d.', 'storefront' ), $i ),
-					'before_widget' 	=> '<aside id="%1$s" class="widget %2$s">',
-					'after_widget' 		=> '</aside>',
-					'before_title' 		=> '<h3>',
-					'after_title' 		=> '</h3>',
-					)
-				);
-			}
-		}
+    public function widgets_init() {
```

Comment:
@jameskoster 

I hope you won't consider my approach to implement filters "over complicated for such a little method"...
But I did start from the principle that users don't need to access the name, id and description of sidebar args. So does it all make sense ?

Not sure to understand what's the problem with the tabulation... is that because we use different tab setting on our editors ?